homeassistant/const.py

```diff
@@ -796,6 +796,14 @@ class UnitOfElectricPotential(StrEnum):
 CURRENCY_CENT: Final = "¢"
 
 
+# Glucose level units
+class UnitOfGlucoseLevel(StrEnum):
+    """Glucose level units."""
+
+    MMOL_L = "mmol/L"
+    MG_DL = "mg/dL"
```

Comment:
Shouldn't this be UnitOfConcentration?

Reply:
Yes - but I forgot to push my commit !!!

Reply:
Good morning!

Reply:
> Yes - but I forgot to push my commit !!!

It's there now: ff2ccd90fcd75c04ddc80589c5650d73110347bf

Note: I've used `UnitOfVolumeConcentration`

Reply:
The consensus from the core team meeting was not to make it concentration by as suggested in the architecture issue: glucose.

Reply:
But they are units of concentration. Which means we can't create a concentration device class because molecular weight, but we can have a glucose device class using units of concentration

Reply:
> But they are units of concentration. Which means we can't create a concentration device class because molecular weight, but we can have a glucose device class using units of concentration

My thinking too...
- Have all concentration units as `UnitOfConcentration`
- Have a device class for `BLOOD_SUGAR_LEVEL` which only allows two units from the enum
- Have a unit converter for `BloodSugarLevelConverter` which only allows those two units

Reply:
> But they are units of concentration. Which means we can't create a concentration device class because molecular weight, but we can have a glucose device class using units of concentration

You where in that meeting and was in that process. I am ok with you raising this again, however, in that case I will revert the decision sooner can re-evaluate that it the next core architecture meeting.

A PR is not the place to have this discussion.

../Frenck

Reply:
> > But they are units of concentration. Which means we can't create a concentration device class because molecular weight, but we can have a glucose device class using units of concentration
> 
> 
> 
> My thinking too...
> 
> - Have all concentration units as `UnitOfConcentration`
> 
> - Have a device class for `BLOOD_SUGAR_LEVEL` which only allows two units from the enum
> 
> - Have a unit converter for `BloodSugarLevelConverter` which only allows those two units

That is not what your architecture proposal described. Do you want to revisit the proposal?

Reply:
Added a comment on https://github.com/home-assistant/architecture/discussions/1146#discussioncomment-10771739